index.js

```diff
@@ -28,22 +28,24 @@ program
       if (err) throw new Error(err);
       var trimRes;
       var results = JSON.parse(body).list;
+      var resultsToDisplay = 3;
+      var magenta = 'ffb3ff';
```

Comment:
put the `#` and `\` in the variable. Also set it to `const` since we won't be changing it.